tests/staggered_invert_test.cpp

```diff
@@ -269,13 +272,13 @@ int main(int argc, char **argv)
 
   // Staggered vector construct START
   //-----------------------------------------------------------------------------------
-  quda::ColorSpinorField *in;
+  std::vector<quda::ColorSpinorField *> in;
```

Comment:
We will also need the `out` fields to be present simultaneously, but I will fix it in the split grid PR.